scripts/set-pusher-suffix.sh

```diff
@@ -6,7 +6,7 @@ CONFIG_FILE="../Web-Expensify/_config.local.php"
 
 if [ -f ".env" ]; then
     # Export vars from the .env file to access the $EXPENSIFY_URL
-    export $(grep -v '^#' .env | xargs)
+    export "$(grep -v '^#' .env | xargs)"
```

Comment:
@roryabraham adding the quotes broke this. Can we make this work and make the linter happy?

Reply:
@neil-marcellini Sorry about that ... created a PR [here](https://github.com/Expensify/App/pull/13497) that should fix it. It's a bit more verbose but in my opinion also a bit clearer what it's doing. We can clean up and merge that PR or remove the quotes and suppress the warning. I'm good with either approach.